example-app/src/main/java/com/adyen/checkout/example/ui/configuration/CheckoutConfigurationProvider.kt

```diff
@@ -99,6 +103,7 @@ internal class CheckoutConfigurationProvider @Inject constructor(
         InstantPaymentConfiguration.Builder(shopperLocale, environment, clientKey)
             .setAmount(amount)
             .setAnalyticsConfiguration(getAnalyticsConfiguration())
+            .setUseSdk(true)
```

Comment:
That's the default value right?

Reply:
Indeed, but I added it for showcase purposes